.github/workflows/cd-staging.yaml

```
@@ -39,21 +43,18 @@ jobs:
           export_default_credentials: true
       - name: Install Python dependencies for deploying
         run: |-
-          pip install -r requirements-deploy.txt
+          poetry install with=dev
```

```suggestion
          poetry install --with=dev
```

Testei aqui https://github.com/basedosdados/pipelines/pull/660

Log https://github.com/basedosdados/pipelines/actions/runs/7920747397/job/21624461290?pr=660#step:6:27

---

Valeu, corrigi abaixo